fastlane/Fastfile

```diff
@@ -0,0 +1,52 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  desc "Submit a new Beta Build to Crashlytics Beta"
+  lane :beta do
+    gradle(task: "clean assembleRelease")
+    crashlytics
+  
+    # sh "your_script.sh"
+    # You can also use other beta testing services here
+  end
+
+  desc "Deploy a new version to the Google Play"
+  lane :deploy do
+    gradle(task: "clean assembleRelease")
+    upload_to_play_store
+  end
+
+  lane :tests do
+    gradle(task: "test")
+  end
+
+  lane :upload_screenshots do
+    upload_to_play_store skip_upload_apk:"true" skip_upload_aab:"true" version_code:"63" skip_upload_changelogs:"true" skip_upload_images:"true" skip_upload_metadata:"true"
+  end
+
+  lane :capture_screen do
+    gradle(task: "clean assembleDebug assembleAndroidTest")
+    capture_android_screenshots
+  end  
```

Comment:
Are we using all the functionality implemented inside this file? Can we remove unused functionality?

Reply:
We use them